README.md

```diff
@@ -415,10 +415,33 @@ Would result in the following schema:
 }
 ```
 
+### Nullable fields, optional fields and compatibility
```

Comment:
Great! Thanks for the good documentation. This is often something that is overlooked when opening up PRs.